tests/platform_tests/test_reboot.py

```diff
@@ -87,22 +87,31 @@ def reboot_and_check(localhost, dut, interfaces, xcvr_skip_list,
     # Append the last reboot type to the queue
     logging.info("Append the latest reboot type to the queue")
     REBOOT_TYPE_HISTOYR_QUEUE.append(reboot_type)
-
+    # wait for dut to come back before checking critical processes/intfs
```

Comment:
perhaps this call of wait_for_startup() is no longer needed after your most recent change?

Reply:
hi @gechiang this is still needed as we discussed. as this is inside tear_down fixture and it's not checking immediately after reboot, we are not passing reboot_type